package.json

```diff
@@ -68,8 +67,14 @@
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
     "ember-source": "^2.15.0",
+    "ember-source-channel-url": "^1.0.1",
+    "ember-try": "^1.0.0-beta.0",
     "loader.js": "^4.2.3"
   },
+  "optionalDependencies": {
+    "ember-cli-changelog": "^0.3.4",
```

Comment:
I've never seen an `optionalDependencies` before, so I think this might not be needed?